frontend/src/api/licensesV3/getActive.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiV3Instance as axios } from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { LicenseV3EventQueueResModel } from 'types/api/licensesV3/getActive';
+
+const getActive = async (): Promise<
+	SuccessResponse<LicenseV3EventQueueResModel> | ErrorResponse
+> => {
+	try {
```

Comment:
Let's remove the try catch block and handle the error in the consuming flow.

Reply:
done